index.html

```diff
@@ -5,232 +5,250 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <title>setup and mobile first</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Desktop version project</title>
 </head>
 <body>
-    <header>
```

Comment:
- [ ] and likewise the header section.

- Your design.
<img width="490" alt="image" src="https://github.com/Dorcas126/Setup-and-mobile-first-project/assets/41163571/b2c392b0-89b0-4740-bb0b-31e697992b96">

- Figma design
<img width="296" alt="image" src="https://github.com/Dorcas126/Setup-and-mobile-first-project/assets/41163571/ded1aa83-b648-4b1c-ad47-dad875167a1e">